Tests/Tests/CustomAttributeSpec.swift

```diff
@@ -116,6 +124,18 @@ class CustomAttributeSpec: QuickSpec {
                     let att2 = CustomAttribute(withKeyName: "1", withTimeInMilliValue: 13333)
                     expect(att1).toNot(equal(att2))
                 }
+
+                it("will return false when object is not CustomAttribute") {
+                    let att1 = CustomAttribute(withKeyName: "CustomAttribute", withStringValue: "value")
+                    let att2 = "Not CustomAttribute"
+                    expect(att1.isEqual(att2)).to(beFalse())
+                }
+
+                it("will return false when objectValue is not NSObject") {
+                    let att1 = CustomAttribute(withKeyName: "CustomAttribute", withStringValue: "value")
+                    var att2 = CustomAttribute(withKeyName: "CustomAttribute", withInvalid: ActionType.invalid)
```

Comment:
So this one is not an NSObject? 🤔

Reply:
This test is to cover `CustomAttribute.isEqual` and it's also comparing the `object.value` Line 64 and Line 65 in this case the `ActionType` was an enum and it is not an NSObject. So, it's returning false.